makefile

```diff
@@ -751,6 +768,6 @@ print-%:
 	@true
 
 # Print the contents of all makefile variables.
-.PHONY: printall
+.PHONY: printall tags
```

Comment:
I'd move this just before the definition of the `tags` target -- keep them next to each other.